src/usr/local/www/services_ntpd.php

```diff
@@ -221,6 +226,11 @@ function build_interface_list() {
 
 init_config_arr(array('ntpd'));
 $pconfig = &$config['ntpd'];
+if (!isset($pconfig['enable']) || ($pconfig['enable'] == 'enabled')) {
+	$pconfig['enable'] = true;
+} else {
+	$pconfig['enable'] = false;
+}
```

Comment:
This could be simplified:

`$pconfig['enable'] = (!isset($pconfig['enable']) || ($pconfig['enable'] == 'enabled'));`